src/styles/_welcome.scss

```
@@ -0,0 +1,11 @@
+.welcome-content {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
+  margin: auto;
+}
+
+.logo {
+  padding: 50px 0;
```

Design shows 40px for vertical padding.
```suggestion
  padding: 40px 0;
```